src/nvim/api/private/helpers.c

```diff
@@ -798,6 +797,11 @@ void api_free_dictionary(Dictionary value)
   xfree(value.items);
 }
 
+void api_free_error(Error value)
```

Comment:
Should we also use a macro version (like api_set_error()) that checks if an error was set before calling this function? In most cases no error should be set.